### PR TITLE
0.8.6 Preflight

### PR DESCRIPTION
**Pull request by wanda-carlson**
0.8.6 Preflight

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This is a good image. 
Screen Shot 2022-01-06 at 2 57 26 PM

**Comment by wanda-carlson** [comment]
**Henry Richard** what do you think about this?

**Comment by henry-richard** [comment]
This is cool!

File: README.md [hunk 1]
```diff
@@ -42,3 +42,4 @@ print("WANDA 3")
 print("WANDA 4") 
 0.8.6 Testing 1
 0.8.6 Testing 1 - continued
```
Comment (henry-richard) [inline README.md:44]: This is old right?

File: README.md [hunk 2]
```diff
@@ -42,3 +42,4 @@ print("WANDA 3")
...
+0.8.6 Preflight
```
Comment (henry-richard) [inline README.md:45]: Nice work!

**Review by henry-richard — DISMISSED** [review]
Minor stuff

**Review by henry-richard — APPROVED** [review]

**Pull request merged**